website/www/site/content/en/blog/capability-matrix.md

```diff
@@ -0,0 +1,604 @@
+---
+title:  "Clarifying & Formalizing Runner Capabilities"
+date:   2016-03-17 11:00:00 -0700
+categories:
+  - beam
+  - capability
+aliases:
+  - /beam/capability/2016/03/17/capability-matrix.html
+authors:
+  - fjp
+  - takidau
+
+capability-matrix-snapshot:
```

Comment:
Would it be possible to keep this in it's separate yaml file and just reference it here rather than in-lining? That would reduce the diff some.

Not a blocker, just a nice-to-have

Reply:
Okay!